source/github-events/on-field-keydown.tsx

```diff
@@ -0,0 +1,28 @@
+import select from 'select-dom';
+import delegate from 'delegate-it';
+
+type DelegateFieldEvent = delegate.EventHandler<KeyboardEvent, HTMLTextAreaElement>;
```

Comment:
Not the correct solution, sadly, but thankfully there's no difference between `textarea` and `input[type=text]`

Reply:
```suggestion
type DelegateFieldEvent = delegate.EventHandler<KeyboardEvent, HTMLTextAreaElement>; // For testing
```

Reply:
What testing?

Reply:
That the commit title works (Commit suggestion)